### DISCUSSION TITLE
Dynamic e-mails using AMP

### DISCUSSION THREAD

**Discussion by iBobik** [Dave136/vue-email#129] [Ideas]
Hello,
are there plans to support https://amp.dev/about/email?

![Snímek obrazovky 2023-10-16 v 20 09 53](https://github.com/Dave136/vue-email/assets/614232/ff6c480f-2db9-4835-93c6-86f2db75403e)

**Comment by Dave136** [comment]
@iBobik hi!

I don't know that tool, it's the first time I see it. But i believe that for the moment is not possible, i'm working on fix library issues.

Any contribution are welcome!

**Comment by DanielRivers** [comment]
I would be up for looking into this.  About to pick up vue-email in a nuxt project.

**Comment by DanielRivers** [comment]
@iBobik Any specific AMP components you have your eye on?

**Comment by iBobik** [comment]
I would like to pull current entity state currently, but I prefer to have at some general docs on how to create AMP version with using native AMP code.

**Comment by Flowko** [comment]
we are not experienced with AMP email, if you can like describe what u need for it to work with vue and vue email, that will help us
